mdal/frmts/mdal_ascii_dat.cpp

```diff
@@ -87,6 +87,11 @@ void MDAL::LoaderAsciiDat::load( MDAL::Mesh *mesh, MDAL_Status *status )
 
   while ( std::getline( in, line ) )
   {
+    //Replace tabs by spaces
+    for(std::string::size_type i = 0; i < line.size(); ++i) {
```

Comment:
please add a small comment that basement 2.8 uses tabs instead of spaces

Maybe use `line = mdal::replace(line, 't', '  ')`?
https://github.com/lutraconsulting/MDAL/blob/master/mdal/mdal_utils.hpp#L45